### PR TITLE
Event log

### PR DESCRIPTION
**Pull request by zuiderkwast**
* Event log
* Remove some debug printouts
* Some simplification of code, renaming stuff, comments, etc.

### CODE REVIEW

File: src/fakeredis_cluster.erl [hunk 1]
```diff
@@ -23,6 +28,7 @@
                , slots_maps = []
                , ask_redirects = #{} :: #{Key :: binary() => {Host :: binary(),
                                                               Port :: inet:port_number()}}
+               , event_log = []      :: [{_, _, _}]
```
Comment (bjosv) [inline src/fakeredis_cluster.erl:31]: is it possible to add name only in spec as doc?, i.e ConnectionId, EventType, Data
ConnectionId and eventtype are known type I guess

**Review by bjosv — COMMENTED** [review]
cool!

File: src/fakeredis_cluster.erl (hunk 1 above)
Comment (zuiderkwast) [reply src/fakeredis_cluster.erl:31]: Yes, I can add that.

You didn't add any types at all to the old fields, so I didn't think it's that important. :-p

Comment (bjosv) [reply src/fakeredis_cluster.erl:31]: not really, but now you started a new trend with ask_redirects :)

**Review by bjosv — APPROVED** [review]

**Pull request merged**